### PR TITLE
Remove exception from check

### PR DESCRIPTION
**Pull request by lukemorton**
http://dev.kohanaframework.org/issues/3868

### CODE REVIEW

File: classes/kohana/orm.php [hunk 1]
```diff
@@ -1228,7 +1228,7 @@ class Kohana_ORM extends Model implements serializable {
 
 		$array = $this->_validation;
 
-		if (($this->_valid = $array->check()) === FALSE OR $extra_errors)
+		if ((($this->_valid = $array->check()) === FALSE OR $extra_errors) && $throw)
```
Comment (ghost) [inline classes/kohana/orm.php]: Use AND instead of &&

Comment (lukemorton) [inline classes/kohana/orm.php]: Fixed.

File: classes/kohana/orm.php [hunk 2]
```diff
@@ -86,7 +86,7 @@ class Kohana_ORM extends Model implements serializable {
 	 */
 	protected static $_properties = array
 	(
-		'object_name', 'object_plural', 'loaded', 'saved', // Object
+		'object_name', 'object_plural', 'loaded', 'saved', 'valid', // Object
```
Comment (lukemorton) [inline classes/kohana/orm.php:89]: Added 'valid' so you can do this:

```
public function action_index()
{
    $post = Model::factory('post')
        ->values(array(
            'title' => 'New',
            'body' => '<p>Body text.</p>',
        ));

    if ( ! $post->check()->valid())
    {
        $this->response->body(Debug::vars($post->validation()->errors()));
        return;
    }

    $post->save();
    $this->response->body(Debug::vars($post));
}
```

**Comment by zeelot** [comment]
This isn't being done in 3.2.0, sorry.

Ticket: http://dev.kohanaframework.org/issues/3763

Someone can go ahead and close this.

**Pull request closed without merging**